docker/Dockerfile.pypy

```diff
@@ -52,7 +53,11 @@ RUN addgroup --gid $B3LBUID b3lb && \
     ln -s $(which pypy3) /usr/local/bin/python3 && \
     ENV_FILE=.env.dev ./manage.py check --force-color && \
     ENV_FILE=.env.dev ./manage.py collectstatic --no-input --force-color
+RUN ./manage.py check --force-color && \
+    ENV_FILE=.env.dev ./manage.py collectstatic --no-input --force-color
 
+RUN mkdir -p /srv/rendering
+RUN chown b3lb:b3lb /srv/rendering
 USER b3lb:b3lb
 ENTRYPOINT ["/usr/src/app/entrypoint.sh"]
-CMD ["uvicorn", "--host", "0.0.0.0", "--ws", "none", "--no-access-log", "--use-colors"]
+CMD ["celery-tasks", "-Q", "b3lb"]
```

Comment:
Is this intentional to replace the default command?

Reply:
Yes, this is intentional, because this image is primary used for celery-tasks.

Reply:
As discussed, changed back to former line, so pypy image can be used as loadbalancer in default configuration.